GDPAPPS-400: implemented presentation framework

mhockenbury opened the pull request:
Updated requirements.txt and setup.py for framework
Implemented presentation framework for Couchbase calls to CORE couchbase

Notes: the updates to the configuration JSON were required to match up to the expectations of the framework since we pass the config object to it to instanitate a CB connection. So the config JSON here is a little messy for now, will need to clean this up a bit, also may be worthwhile to match up our naming conventions for akira's CB stuff with framework (eg. multiget vs get_multi)

~~TODO: still need to dynamically get different asset formats (currently hardcoded to 'full')~~

alon85 commented on appserv_akira/application.py:
```diff
@@ -29,14 +29,14 @@
 
 
-def get_user_feed(userId, field_list=None):
+def get_user_feed(userId, field_list=None, transform='full'):
```
write a TODO to change this to default to assetLite and make configurable

alon85 commented:
Take out "_asset" since it is taken care of automatically in framework.

alon85 commented:
You're missing a coma in your setup.py after "APScheduler"

mhockenbury commented:
@alon85 I took out the "asset_" prefix I was adding, fixed the comma, and implemented loading configs by setting the `ENVIRONMENT`env var to the appropriate value (ex. local, development)
